d2l-input-checkbox.js

```diff
@@ -179,11 +191,23 @@ Polymer({
 	},
 	_handleChange: function(e) {
 		this.checked = e.target.checked;
+		this.indeterminate = false;
 		this.dispatchEvent(new CustomEvent(
 			'change',
 			{bubbles: true, composed: false}
 		));
 	},
+	_onClick: function() {
```

Comment:
Nit: I might name the handler consistent with the others here.  `_handleClick`

Reply:
^ will change